index.html

```diff
@@ -0,0 +1,474 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
+    <!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"> -->
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <div class="header">
+            <nav class="flex-container">
+                <div class="flex-container">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-list" viewBox="0 0 16 16">
+                        <path fill-rule="evenodd" d="M2.5 11.5A.5.5 0 0 1 3 11h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4A.5.5 0 0 1 3 7h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4A.5.5 0 0 1 3 3h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"/>
+                    </svg>
+                    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-search" viewBox="0 0 16 16">
+                        <path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/>
+                    </svg>
+                    <span class="site-name">space & cosmos</span>
+                </div>
+                <div>
+                    <!-- <img src="images/thenewyourtimes.png" alt="The New York Times" width="213" height="31"> -->
+                    <h2 class="site-logo">The New York Times</h2>
+                </div>
+                <div class="flex-container">
+                    <button class="subscribe" type="button">Subscribe now</button>
+                    <button type="button">Account</button>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <main>
+        <article>
+            <h1 style="font-size: 0;">Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+            <section class="advert">
+                <h2>Advert Space</h2>
+            </section>
+            <section class="text headline">
+                <h2>Space Ripples Reveal Big Bang’s Smoking Gun</h2>
```

Comment:
## Improve the styling on the `Space Ripples Reveal Big Bang’s Smoking Gun` section
- [ ] Kindly remove the spacing in between the main image of the `Space Ripples Reveal Big Bang’s Smoking Gun` article and the heading itself as it is not supposed to be there.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/francisuloko/new-york-times-clone/add-article/index.html)|
|:--:|:--:|
|![main-image](https://user-images.githubusercontent.com/22345961/108378250-e2ce9080-7215-11eb-8e20-ca3a5c8963bf.png)|![main-image-clone](https://user-images.githubusercontent.com/22345961/108378274-e6faae00-7215-11eb-87a4-2cf820874822.png)|